cmd/common.go

```diff
@@ -28,14 +29,36 @@ func syscallExec(binaryFile string, args ...string) error {
 }
 
 func ExecBinScript(script string, args ...string) error {
-	path := filepath.Join(
-		os.Getenv("GOPATH"),
-		"src/github.com/metacurrency/holochain",
-		"bin",
-		script)
+	path := GolangHolochainDir("bin", script)
+  fmt.Printf("HC: common.go: ExecBinScript: %v (%v)", path, args)
 	return syscallExec(path, args...)
 }
 
+func OsExecSilent(args ...string) error {
+  cmd := exec.Command(args[0], args[1:]...)
+  fmt.Printf("common.go: OsExecSilent: %v", cmd)
```

Comment:
my preference would be to have these kinds of printfs either be only printed in verbose mode, or maybe even in debug mode because they are referencing a code file.  It does make sense to me to add generalized "verbose" and or logging support for commands into common.go.  Note that I've done a bunch of extracting of the logging code in holochain to it's own repo/library (see https://waffle.io/metacurrency/holochain/cards/58fe20ada157b1004a82f49f) which if we completed (hint hint), we could then also re-use in cmd/common.go